server/api/repo.go

```diff
@@ -150,7 +151,8 @@ func PostRepo(c *gin.Context) {
 
 	// creates the jwt token used to verify the repository
 	t := token.New(token.HookToken)
-	t.Set("repo-id", strconv.FormatInt(repo.ID, 10))
+	t.Set("forge-remote-id", string(forgeRemoteID))
```

Comment:
`forge-remote-id` was a bit confusing for me in the first moment as it does not tell its the repos `forge-remote-id`, maybe sth like `repo-forge-remote-id` would be better.